packages/web/src/scss/components/Stack/_Stack.scss

```diff
@@ -32,20 +32,14 @@
     border-block-start: theme.$border;
 }
 
-.Stack--hasSpacing > * {
-    @include spacing.prevent-inheritance(
-        $input-property-base-names: '--stack-spacing',
-        $breakpoints: theme.$breakpoints
-    );
-}
-
+// stylelint-disable custom-property-pattern -- Ignore for generated private custom properties.
 .Stack--hasSpacing.Stack--hasStartDivider > *,
 .Stack--hasSpacing.Stack--hasEndDivider > * {
-    padding-block: calc(var(--stack-gap) / 2);
+    padding-block: calc(var(--_stack-spacing) / 2);
```

Comment:
I was happy about the `_` prefix until I ran across this use case. 🤔 Should the "internal" custom property name be different so Stylelint doesn't need to be turned off?

Reply:
What about just using `internal_` as a prefix so it would be clear for anyone?

Reply:
Agreed we will go with the `internal` suffix for both internal custom properties (this and the default one).